Makefile

```diff
@@ -5,9 +5,9 @@ PREFIX?=$(shell pwd)
 VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
 GO_LDFLAGS=-ldflags "-X `go list ./version`.Version $(VERSION)"
 
-.PHONY: clean all fmt vet lint build test binaries
+.PHONY: clean all fmt vet lint deps build test binaries
 .DEFAULT: default
-all: AUTHORS clean fmt vet fmt lint build test binaries
+all: AUTHORS clean fmt vet fmt lint deps build test binaries
```

Comment:
I am not sure the `deps` target should always be run. Please see my other comment for details.